internal/testrunner/runners/system/runner.go

```diff
@@ -105,53 +105,28 @@ func (r *runner) TearDown() error {
 	return nil
 }
 
-type resultComposer struct {
-	testrunner.TestResult
-	startTime time.Time
-}
-
-func (r *runner) newResult(name string) resultComposer {
-	return resultComposer{
+func (r *runner) newResult(name string) testrunner.ResultComposer {
+	return testrunner.ResultComposer{
 		TestResult: testrunner.TestResult{
 			TestType:   TestType,
 			Name:       name,
 			Package:    r.options.TestFolder.Package,
 			DataStream: r.options.TestFolder.DataStream,
 		},
-		startTime: time.Now(),
+		StartTime: time.Now(),
```

Comment:
Maybe a constructor which will set the startTime? I saw you set this value in multiple places.

Reply:
Done in d93be29.